pkg/handlers/handlers.go

```diff
@@ -292,9 +292,14 @@ func (h Handlers) getByName(rw http.ResponseWriter, req *http.Request) {
 
 	name := unquote(req.FormValue("name"))
 
+	var hidden bool
+	if value := req.FormValue("hidden"); value != "" {
+		hidden = value == "true"
```

Comment:
Is it expected here not to use https://pkg.go.dev/strconv#ParseBool?
Maybe the form parameter should also be called `filterHidder`, wdyt?